sections/glossary.txt

```diff
@@ -381,6 +381,12 @@ the definitions for more information.
 	the [[#conf-interactive-ua]] and [[#conf-author-tools]] for details on the
 	requirements for a <em>conforming</em> user agent.
 
+: <dfn>wheel event group</dfn>
```

Comment:
We're moving away from having entries in glossary since they are not normative and mostly duplicate definitions that are properly defined elsewhere.

I think this should just be included up with the rest of the text.